apps/wallet/src/ui/app/components/network-selector/index.tsx

```diff
@@ -46,30 +61,46 @@ const NetworkSelector = () => {
         <div className={st.networkOptions}>
             <ul className={st.networkLists}>
                 {netWorks.map((apiEnv) => (
-                    <li
-                        className={st.networkItem}
-                        key={apiEnv.networkName}
-                        data-network={apiEnv.networkName}
-                        onClick={changeNetwork}
-                    >
-                        <Icon
-                            icon="check2"
-                            className={cl(
-                                st.selectedNetwork,
-                                selectedApiEnv === apiEnv.networkName &&
-                                    st.networkActive
-                            )}
-                        />
-                        <div style={apiEnv.style}>
+                    <li className={st.networkItem} key={apiEnv.networkName}>
+                        <Link
```

Comment:
This shouldn't be a link. It should be a `button` with `type="button"`.

Reply:
done